frontend/src/App.js

```diff
@@ -9,6 +9,7 @@ import RestaurantCreatePage from "main/pages/Restaurants/RestaurantCreatePage";
 import RestaurantEditPage from "main/pages/Restaurants/RestaurantEditPage";
 import RestaurantIndexPage from "main/pages/Restaurants/RestaurantIndexPage";
 import RestaurantDetailsPage from "main/pages/Restaurants/RestaurantDetailsPage";
+import LaptopCreatePage from "main/pages/Laptops/LaptopCreatePage";
```

Comment:
don't need to add anything to main yet https://ucsb-cs156.github.io/s23/lab/team01/team01_6e.html

Reply:
The TA did this when I worked on it regarding 6e. I can remove it for the merge conflict though.